### PR TITLE
Improve PHP Inline Documentation

### PR DESCRIPTION
**Pull request by adamsilverstein**
### Issue Number
Work for core merge, see https://github.com/GoogleChromeLabs/wp-sitemaps/issues/164.

### Description
A summary of the change. The summary should include what problem is addressed and how it has been fixed.

### Screenshots (before and after if applicable)
If your PR includes visual changes include before and after screenshots showing the change.

### Type of change
Please select the relevant options:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Enhancement (change which improves an existing feature. E.g., performance improvement, docs update, etc.)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Steps to test
Describe the tests required to verify your changes.
Provide instructions so the PR Tester can check functionality and also list any relevant details and / or dependencies required for your tests.

### Acceptance criteria
- [x] My code follows WordPress coding standards.
- [x] I have performed a self-review of my own code.
- [ ] If the changes are visual, I have cross browser / device tested.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [ ] I have added test instructions that prove my fix is effective or that my feature works.

### CODE REVIEW

File: inc/class-core-sitemaps-provider.php [hunk 1]
```diff
@@ -39,7 +47,9 @@ public function get_url_list( $page_num, $type = '' ) {
 	}
 
 	/**
-	 * Return object type being queried.
+	 * Returns object type being queried.
```
Comment (adamsilverstein) [inline inc/class-core-sitemaps-provider.php]:
```suggestion
	 * Returns the object type being queried.
```

File: inc/class-core-sitemaps-renderer.php [hunk 2]
```diff
@@ -83,18 +97,22 @@ public function get_sitemap_index_stylesheet_url() {
 		}
 
 		/**
-		 * Filter the URL for the sitemap index stylesheet.
+		 * Filters the URL for the sitemap index stylesheet.
```
Comment (adamsilverstein) [inline inc/class-core-sitemaps-renderer.php:100]: This filter doc block is a duplicate and should reference the first.

File: readme.txt [hunk 3]
```diff
@@ -2,7 +2,7 @@
 Contributors: joemcgill, pacifika, kburgoine, tweetythierry, swissspidy
 Tags: seo, sitemaps
 Requires at least: 5.3
```
Comment (swissspidy) [inline readme.txt]: Let's also bump this to 5.4 while we're at it

**Review by swissspidy — APPROVED** [review]

File: readme.txt (hunk 3 above)
Comment (adamsilverstein) [reply readme.txt]: 👍

Comment (adamsilverstein) [reply readme.txt]: Bumped in https://github.com/GoogleChromeLabs/wp-sitemaps/pull/173/commits/5fadaecb87c0822660e05241f3eaf65e6a169824

File: inc/functions.php [hunk 4]
```diff
@@ -57,7 +63,9 @@ function core_sitemaps_get_server() {
 }
 
 /**
- * Get a list of sitemaps.
+ * Gets a list of sitemaps.
```
Comment (felixarntz) [inline inc/functions.php]: Maybe "Gets a list of sitemap providers." instead? That's a bit more precise and in line with the description for the other function below.

**Review by felixarntz — APPROVED** [review]

File: inc/functions.php (hunk 4 above)
Comment (adamsilverstein) [reply inc/functions.php]: 👍

Comment (adamsilverstein) [reply inc/functions.php]: Updated in https://github.com/GoogleChromeLabs/wp-sitemaps/pull/173/commits/c2d3d4b4f1fed1ad34e041d9164c6bf071fb1d8e.

**Pull request merged**